app/helpers/user_organizations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UserOrganizationsHelper
```

Comment:
Remove empty helpers - as a general rule of thumb, remove files which are empty while they are generated.